examples/mujoco/fetch_her_ddpg.py

```diff
@@ -30,7 +29,7 @@
 
 def get_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--task", type=str, default="FetchReach-v3")
+    parser.add_argument("--task", type=str, default="FetchReach-v2")
```

Comment:
why this change?

Reply:
There is no v3 in gymnasium robotics (https://robotics.farama.org/envs/fetch/)